### PR TITLE
Census communal increase test coverage of downstream data

### PR DESCRIPTION
**Pull request by yoganandkunche**
What is the context of this PR?

Dumps the answers that are going to be submitted and compares with the answers that were entered for Census_Communal

How to review

Run the integration test.

### CODE REVIEW

**Comment by dcdarrell9** [comment]
Is it worth having different answers for `expected_downstream_data` 
Or is there no need?

**Comment by yoganandkunche** [comment]
@dcdarrell9 we are only verifying submission 'data' for census schemas.

File: tests/integration/census/test_census_communal_submission_data.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+class TestCensusCommunalSubmissionData(IntegrationTestCase):
+
+    def test_census_communal_data_matches_census_communal(self):
+        self.complete_survey('census', 'communal')
+
+        # Only verifying 'data'
+        actual_downstream_data = self.dumpSubmission()['submission']['data']
+        expected_downstream_data = self.get_expected_submission_data()
+
+        self.assertCountEqual(actual_downstream_data, expected_downstream_data)
```
Comment (jonnyshaw89) [inline tests/integration/census/test_census_communal_submission_data.py:14]: This is only testing it has the same number of answers and not that the data in actually what was entered?

Comment (yoganandkunche) [reply tests/integration/census/test_census_communal_submission_data.py:14]: It also verifies the data entered.

Comment (jonnyshaw89) [reply tests/integration/census/test_census_communal_submission_data.py:14]: So it does.

**Review by jonnyshaw89 — APPROVED** [review]

**Review by dcdarrell9 — APPROVED** [review]

**Pull request merged**